fonts.css

```diff
@@ -1,82 +1,410 @@
-@font-face {
-	font-family: "Montserrat";
-	src: url("C:\Users\пк\Desktop\Fonts\montserrat-bold-webfont.woff2") format("woff2"),
-		 url("C:\Users\пк\Desktop\Fonts\montserrat-bold-webfont.woff") format("woff");
-	font-weight: 700;
-	font-style: normal;
-}
-
-@font-face {
-	font-family: "Montserrat";
-	src: url("C:\Users\пк\Desktop\Fonts\montserrat-medium-webfont.woff") format("woff"),
-		 url("C:\Users\пк\Desktop\Fonts\montserrat-medium-webfont.woff2") format("woff2");
-	font-weight: 500;
-	font-style: normal;
-}
-
-@font-face {
-    font-family: "Montserrat";
-    src: url("C:\Users\пк\Desktop\Fonts\montserrat-semibold-webfont.woff2") format("woff2"),
-         url("C:\Users\пк\Desktop\Fonts\montserrat-semibold-webfont.woff") format("woff");
-    font-weight: 600;
-    font-style: normal;
-}
+/* Шрифты */
+	@font-face {
+		font-family: "Montserrat";
+		src:	url("Fonts/Montserrat-Bold.otf") format("opentype"),
+					url("Fonts/Montserrat-Bold.ttf") format("truetype");
+		font-weight: 700;
+		font-style: normal;
+	}
+
+	@font-face {
+		font-family: "Montserrat";
+		src: url("Fonts/Montserrat-Medium.otf") format("opentype"),
+				 url("Fonts/Montserrat-Medium.ttf") format("truetype");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)